packages/gatsby-plugin-image/src/transforms/__testfixtures__/gatsby-plugin-image/mansion.output.js

```diff
@@ -0,0 +1,74 @@
+import { graphql } from "gatsby"
```

Comment:
Is this checking that it's unchanged then?

Reply:
Yes